### PR TITLE
refactor: useCategoryTree

### PR DESCRIPTION
**Pull request by WojtekTheWebDev**
Refactor `useCategoryTree`

## Description

Remove id param from `useCategoryTree`

## Motivation and Context

Category tree in BigCommerce is always the same, I don't want to differentiate it by `id` if it always would respond wit the same category tree items

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by Mateuszpietrusinski — COMMENTED** [review]
What is the purpose of that change, you are changing this sharedRef to one key so it will be singleton after it

File: packages/composables/src/useCategoryTree/index.ts [hunk 1]
```diff
@@ -11,32 +11,29 @@ import { params } from './params';
 /**
  * Managing category lists (but not category products).
  */
-export const useCategoryTree = (id: string): UseCategoryTreeResponse => {
+export const useCategoryTree = (): UseCategoryTreeResponse => {
   const context = generateContext(params) as Context;
 
   const categoryTreeItems: Ref<CategoryTree[]> = sharedRef(
```
Comment (Frodigo) [inline packages/composables/src/useCategoryTree/index.ts:17]: @WojtekTheWebDev do you plan to remove sharedRefs and generateContext function in near future?

Comment (WojtekTheWebDev) [reply packages/composables/src/useCategoryTree/index.ts:17]: There is a plan to move composables to `theme` dir so yes, but it would be probably in april

**Review by Frodigo — APPROVED** [review]

**Comment by WojtekTheWebDev** [comment]
@Mateuszpietrusinski Category tree in BigCommerce is always the same, I don't want to differentiate it by `id` if it always would respond wit the same category tree items

**Pull request merged**